docs/tutorial/tutorial_custom.rst

```diff
@@ -56,11 +61,55 @@ object directly and the funtions are not allowed to return any information.
 
        return
 
-Attaching Custom Function
-^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Custom methods must be attached to an :py:class:`Instrument` object for pysat
-to automatically apply the method upon every load.
+Custom functions can also be applied at the :py:class:`Constellation` level,
+which allows data from multiple :py:class:`Instrument` objects to be used by
+a single function.  :py:class:`Constellation` level custom functions are applied
+after :py:class:`Instrument` level functions.  Note that the Instrument loading
+order for this function is important.  An alternate method would be to identify
```

Comment:
Looks to me that all Instruments are loaded and then the constellation functions are applied. I don't understand how load order is important.

Reply:
Ah, the use of the index to access the relevant instrument. I'd suggest putting that point in there since the followup comment now makes more sense to me.